vignettes/editors.Rmd

```diff
@@ -12,13 +12,21 @@ knitr::opts_chunk$set(
   collapse = TRUE,
   comment = "#>"
 )
+
+maybe_still <- function(url) {
+  if (pkgdown::in_pkgdown()) {
+    url
+  } else {
+    gsub("\\.gif$", "-still.gif", url)
+  }
+}
 ```
 
 ## RStudio
```

Comment:
WDYT about a note at the top here that directs users to the pkgdown version to see full animation?

Reply:
Not sure how useful this is to a user browsing `vignette("editors")`. Maybe instead add a note to `?lintr-package`?

Reply:
I do think it's quite useful.

> Note: this vignette is best viewed in the browser, where we can render full animations of editor flows. See <link>.

?